zam/aspirateur/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+
+setup(
+    name='zam-aspirateur',
```

Comment:
Alright, I'll go that way with the client, I wasn't sure how we want to do this.